sowm.c

```diff
@@ -265,8 +276,8 @@ int main(void) {
 
```

Comment:
int s is already declared globally can you remove the 'int' in main method so 's' can be accessed in your getcolor method